test/integration/f22/Makefile

```diff
@@ -36,4 +36,4 @@ remote_sync: remote_setup
 	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch && make"
 
 remote_slow: remote_sync
-	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make slow"
+	ssh $(SSH_USER)@$(SSH_HOST) "cd kpatch-test/kpatch/test/integration/f22 && make local"
```

Comment:
The target is "remote_slow" so why not "make slow"?  I get that the local target requires the slow target and so the end result is the same.  Just wondering what the reason for the change was.